command/httlemon.py

```diff
@@ -1,12 +1,32 @@
 import click
 
 from client.client_request import client_request
+from httlemon import collections
 
+from functools import update_wrapper
 
-@click.command()
+
+@click.group()
+def httlemon():
+    pass
+
+@httlemon.command()
 @click.argument('http_verb')
 @click.argument('url')
-def httlemon(http_verb, url):
+def req(http_verb, url):
```

Comment:
why `req`? In my head you should just be able to run `httlemon get <something else>` / `httlemon post <something else>` or whatever.